app/controllers/api/generic_object_definitions_controller.rb

```diff
@@ -1,5 +1,11 @@
 module Api
   class GenericObjectDefinitionsController < BaseController
+    include Api::Mixins::GenericObjects
```

Comment:
In short, I think you needed the `Api` qualification because the classic UI has a top-level `Mixins` namespace. If something in there gets loaded first, it will break the autoloading of this module. I'm proposing that the classic UI not do that and put their mixins in their own namespace.

Reply:
thanks @imtayadeway !!! ✨

Reply:
Or you could use `include_concern Mixins::GenericObjects` which I guess is the standard way that we deal with this problem (and may convey to the next person that there is some kind of contention over the `Mixins` namespace)

Reply:
@imtayadeway unfortunately I tried this, and the API is not happy :(